lib/item-comments.js

```diff
@@ -0,0 +1,22 @@
+var Sequelize = require('sequelize');
+
+module.exports = function(db) {
+  var ItemComments = db.define('item_comments', {
+    id: {
+      type: Sequelize.INTEGER,
+      autoIncrement: true,
+      primaryKey: true
+    },
+    body: {
+      type: Sequelize.TEXT
```

Comment:
I think `body` should be a required field, which it sounds like we can enforce by adding `allowNull: false`. What do you think?